spec/features/course/assessment_condition_management_spec.rb

```diff
@@ -63,8 +69,10 @@
         expect do
           find_link(nil, href: condition_delete_path).click
         end.to change { assessment.conditions.count }.by(-1)
+        title = I18n.t('activerecord.attributes.course/condition/assessment/title.complete',
+                       assessment_title: other_assessment.title)
```

Comment:
The tests will still pass is because we have an extension which let rails only shows keys in test env, by right it should be there.  but I am also fine with both as long as the test passed.